src/poseidon_gencontract.js

```diff
@@ -9,92 +9,91 @@ import { ethers } from "ethers";
 
```

Comment:
improvement: several changes are due to formatting only. to avoid this, we can add a formatter as pre-commit hooks, and check in the CI that the format is respected. that way, we'll only have meaningful changes in a PR, and no longer changes related to format.

Reply:
Yes that a bit sloppy I think, apologies, that formatting was giving me a headache. So I went ahead with the formatting change. But I should have kept them as separate PRs. I will point out the lines that changed

Reply:
ok no pb, this change be improved in another PR. it's just a comment to improve readability, it's not a real problem to address